hack/remove-calico-policy/README.md

```diff
@@ -0,0 +1,60 @@
+# Disabling and Removing Calico Policy
+
+This guide explains how to disable and remove Calico policy from a running cluster.  These steps are intended as a
+last resort should it appear that Calico policy is not functioning properly on a production system.
+
+The steps in this directory are specific to clusters running on GCE using the
+Kubernetes `cluster/kube-up.sh` script, but can be generalized to other Calico deployments.
+
+Calico policy can be disabled and removed for troubleshooting purposes or in emergency situations using the following steps.
+
+### Requirements / Assumptions
+
+- Calico version v2.2 or higher
+- Kubernetes v1.6 or higher
+- These steps assume Calico is running in policy-only mode (without BGP networking)
```

Comment:
> without BGP networking

I think just saying `without networking` is enough, BGP is just a backend implementation. `BGP networking` sounds confusing to someone not familiar with how Calico networking backend works